src/PhpSpec/Formatter/CurrentExampleWriter.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of PhpSpec, A php toolset to drive emergent
+ * design by specification.
+ *
+ * (c) Marcello Duarte <marcello.duarte@gmail.com>
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PhpSpec\Formatter;
+
+use PhpSpec\Console\IO;
+use PhpSpec\Message\CurrentExample;
+
+class CurrentExampleWriter
```

Comment:
It might make sense to introduce an interface here (and finalise the class) - we may want to present fatals in different ways in future.